servers/rendering/renderer_rd/storage_rd/light_storage.h

```diff
@@ -544,6 +544,29 @@ class LightStorage : public RendererLightStorage {
 		return light->shadow;
 	}
 
+	virtual bool light_is_shadow_visible_at_position(RID p_light_instance, const Vector3 position) const override {
+		const LightInstance *light_instance = light_instance_owner.get_or_null(p_light_instance);
+		ERR_FAIL_NULL_V(light_instance, RS::LIGHT_DIRECTIONAL);
```

Comment:
This error seems to be copy-pastad quite a bit prior to your PR.
`ERR_FAIL_NULL_V` should return true or false I think, not `RS::LIGHT_DIRECTIONAL`.

Reply:
Classic copy paste without reading. 😅  Surprised that didn't end up with a compiler error.